### PR TITLE
Added a Gazebo layer and more dependencies

### PR DESCRIPTION
**Pull request by Isopod00**

### CODE REVIEW

File: docker/Dockerfile.umn [hunk 1]
```diff
@@ -18,8 +18,16 @@ RUN apt-get update  && apt-get install -y \
     ros-humble-grid-map-rviz-plugin \
     ros-humble-can-msgs \
     ros-humble-usb-cam \
+    python3-pytest \
+    python3-opencv \
+    python3-serial \
+    python3-numpy \
+    libboost-all-dev \
+    ros-humble-ros-gz \
```
Comment (jmblixt3) [inline docker/Dockerfile.umn]: Move this to gazebo layer

File: docker/Dockerfile.umn [hunk 2]
```diff
@@ -18,8 +18,16 @@ RUN apt-get update  && apt-get install -y \
     ros-humble-grid-map-rviz-plugin \
     ros-humble-can-msgs \
     ros-humble-usb-cam \
+    python3-pytest \
+    python3-opencv \
+    python3-serial \
+    python3-numpy \
+    libboost-all-dev \
```
Comment (jmblixt3) [inline docker/Dockerfile.umn:25]: I'm curious what this depends on it's quite bulky, is it can?

File: docker/Dockerfile.umn [hunk 3]
```diff
@@ -18,8 +18,16 @@ RUN apt-get update  && apt-get install -y \
     ros-humble-grid-map-rviz-plugin \
     ros-humble-can-msgs \
     ros-humble-usb-cam \
+    python3-pytest \
+    python3-opencv \
```
Comment (jmblixt3) [inline docker/Dockerfile.umn:22]: Should this be moved to real sense layer or smth

File: docker/Dockerfile.umn [hunk 4]
```diff
@@ -18,8 +18,16 @@ RUN apt-get update  && apt-get install -y \
...
+    python3-serial \
+    python3-numpy \
+    libboost-all-dev \
+    ros-humble-ros-gz \
+    ros-humble-sdformat-urdf \
+    ros-humble-joint-state-publisher-gui \
+    python3-tqdm \
+    python3-requests \
+    libeigen3-dev \
+    ros-humble-librealsense2 \
```
Comment (jmblixt3) [inline docker/Dockerfile.umn:29]: Why is this not in real sense layer?

**Review by jmblixt3 — CHANGES_REQUESTED** [review]
Fix the comments pls

File: docker/Dockerfile.umn (hunk 2 above)
Comment (Isopod00) [reply docker/Dockerfile.umn:25]: Yes, the CAN bridge depends on libboost-all-dev

File: docker/Dockerfile.umn (hunk 4 above)
Comment (Isopod00) [reply docker/Dockerfile.umn:29]: idk man, you tell me

Comment (Isopod00) [reply docker/Dockerfile.umn:29]: oh its prob because skimmer node depends on realsense

Comment (jmblixt3) [reply docker/Dockerfile.umn:29]: Ohhhh, can it be decoupled :(

Comment (Isopod00) [reply docker/Dockerfile.umn:29]: eventually yeah, but it's probably fine for now

File: docker/Dockerfile.umn (hunk 3 above)
Comment (Isopod00) [reply docker/Dockerfile.umn:22]: I think we need opencv anyways for apriltags and maybe nvblox? i don't think it is only for realsense but I could be wrong

**Pull request merged**